CMakeLists.txt

```diff
@@ -533,6 +533,18 @@ if(H3_IS_ROOT_PROJECT AND BUILD_TESTING)
         endif()
     endmacro()
 
+    macro(add_h3_cli_test name h3_args expect_string)
+        add_test(NAME ${name}_test${test_number}
+                 # /usr/bin/sh -c "test \"`./bin/h3 cellToLatLng -c 8928342e20fffff`\" = \"37.5012466151, -122.5003039349\""
+                 COMMAND ${SHELL} "test \"`./bin/h3 ${h3_args}`\" = '${expect_string}'")
+
+        if(PRINT_TEST_FILES)
+            message("${name}_test${test_number} - ${argfile}")
+        endif()
+
+        # No coverage possible for H3 CLI testing
```

Comment:
I assume coverage should be somehow possible (would need to configure the binaries to produce coverage, and include them in the considered source paths)?

Reply:
I'll study it, but okay if that's a separate PR?